build.include

```diff
@@ -36,6 +37,9 @@ parse() {
       --push)
         PUBLISH_IMAGES=true
       shift ;;
+      --push-to-quay)
```

Comment:
what's the difference between push and push-to-quay ?
AFAIK it should be the same thing

Reply:
reworked